renderdoc/replay/replay_driver.cpp

```diff
@@ -565,7 +565,7 @@ bool HighlightCache::FetchHighlightPositions(const MeshDisplay &cfg, FloatVector
   {
     uint32_t dim = PatchList_Count(cfg.position.topo);
 
-    uint32_t v0 = uint32_t(idx / dim) * dim;
+    uint32_t v0 = dim == 0 ? 0 : (uint32_t(idx / dim) * dim);
```

Comment:
I see. I am not too familiar with how `PatchList_Count` is supposed to work.
I've implemented the `+ 1` in the latest revision.